### PR TITLE
Add GHA test for find_package integration.

### PR DESCRIPTION
**Pull request by alexreinking**
Adds a CI test for #2446 to #2447

This is a PR to a PR, not to `main`.

ATTN @keithw @steven-johnson

### CODE REVIEW

**Review by steven-johnson — APPROVED** [review]

**Comment by alexreinking** [comment]
Looks like it's working now.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -162,10 +162,35 @@ jobs:
...
         cmake --version
-    - name: Configure WABT
+
+    - name: configure (OpenSSL)
       run: cmake -G Ninja -S . -B out -DCMAKE_BUILD_TYPE=Release
-    - name: build
+    - name: build (OpenSSL)
       run: cmake --build out
+    - name: install (OpenSSL)
+      run: cmake --install out --prefix _opt/wabt-openssl
```
Comment (sbc100) [inline .github/workflows/build.yml:171]: What is `_opt` here?  How about `install/...`?

Comment (sbc100) [reply .github/workflows/build.yml:171]: Or how about `install` and install-picocha` to match the `out` names?

Comment (alexreinking) [reply .github/workflows/build.yml]: Done!

**Comment by alexreinking** [comment]
@sbc100 - thanks for the review! We can merge this into #2447 any time and then turn our attention to the overall patch.

**Comment by alexreinking** [comment]
@steven-johnson -- do you have the power to click "merge" on this PR?

**Pull request merged**